src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirective.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
     public static class InjectDirective
     {
-        public static readonly DirectiveDescriptor Directive = DirectiveDescriptorBuilder.Create("inject").AddType().AddMember().Build();
+        public static readonly DirectiveDescriptor Directive = DirectiveDescriptor.CreateDirective(
+            "inject",
+            DirectiveDescriptorKind.SingleLine,
+            builder => builder.AddTypeToken().AddMemberToken());
```

Comment:
Isn't that going back to the CreateSingleLineDirective/CreateRazorBlockDirective etc. approch?